src/cascadia/TerminalConnection/EchoConnection.h

```diff
@@ -5,23 +5,23 @@
 
 #include "EchoConnection.g.h"
 
+#include "../cascadia/inc/cppwinrt_utils.h"
```

Comment:
at what point should this just be precomp'd?

Reply:
Good question :)